src/parse/asp/builtins.go

```diff
@@ -155,6 +159,57 @@ func buildRule(s *scope, args []pyObject) pyObject {
 	return pyString(":" + target.Label.Name)
 }
 
+// doExec fork/exec's a command and returns the output as a string.  exec
+// accepts either a string or a list of commands and arguments.
+func doExec(s *scope, args []pyObject) pyObject {
+	cmdIn := args[0]
+	wantStdout := args[1].IsTruthy()
+	wantStderr := args[2].IsTruthy()
```

Comment:
We should probably explicitly error if these are both set to False, as it will cause a fall-through in the switch below and the attempted execution of a command with no required result (which can only mean people have made a mistake or are trying to do unprincipled "bad things").